unstructured_inference/models/chipper.py

```diff
@@ -1068,8 +1230,27 @@ def __call__(
         return False
 
 
+class TargetTokenIdStoppingCriterion(StoppingCriteria):
+    def __init__(self, target_token_id):
+        super().__init__()
+        self.target_token_id = target_token_id
+
+    def __call__(
+        self,
+        input_ids: torch.LongTensor,
+        scores: torch.FloatTensor,
+        **kwargs,
+    ) -> bool:
+        """
+        Check if the already generated tokens contain the target token_id
+        """
+        output = self.target_token_id in input_ids
+
+        return output  # self.target_token_id in input_ids
+
+
 def _no_repeat_ngram_logits(
-    input_ids: torch.LongTensor,
+    input_ids: torch.Tensor,
```

Comment:
the input is just sliced but from the same tensor as before, right? So the datatype should stay as `LongTensor`?